pkg/specgen/generate/container_create.go

```diff
@@ -245,9 +245,12 @@ func createContainerOptions(ctx context.Context, rt *libpod.Runtime, s *specgen.
 		if err != nil {
 			return nil, err
 		}
-		if newWD != "" {
-			options = append(options, libpod.WithCreateWorkingDir())
-		}
+		s.WorkDir = newWD
```

Comment:
I don't think we should be modifying the specgen here - maybe in CreateContainer, not createContainerOptions?

Reply:
The problem is WithCreateWorkingDir works off of that field.  Or we have to do the check twice.

Reply:
Ok reworked it placing the WorkDir check into CreateContainer.